resources/views/components/user/profile/moderation-tools.blade.php

```diff
@@ -0,0 +1,134 @@
+@props([
+    'targetUser' => null, // User
+])
+
+<?php
+use App\Community\Enums\ArticleType;
+use App\Community\Enums\UserAction;
+use App\Enums\Permissions;
```

Comment:
To match the other components after the refactoring, these use statements should be at the very top of the file.

Reply:
Fixed in latest.

While working on this, I couldn't help but notice that a `@use()` directive has been added to Laravel recently:

```php
@use('App\Community\Enums\ArticleType')
@use('App\Community\Enums\UserAction')
@use('App\Enums\Permissions')
```

Is this permissible in RAWeb?

Reply:
I think that'd mess with the variety of editors we have in use as those wouldn't properly detect those as imports. Does VS Code understand those directives?
Personally, I'd prefer native php tags at the very top for now 🤔

Reply:
In regards to a potential introduction of bladestan or similar this would also mean that  eventually this php block should contain `declare(strict_types=1);` which i don't think blade has an equivalent to. At least i couldn't find anything on that after a quick search.

Reply:
Looking into `@use()` more on a machine where have more IDEs than Sublime Text available, it seems not even PHPStorm with the paid Laravel Idea plugin handles those properly. 
That'd render PHPStorm's refactoring features slightly less useful and pose some additional risks if an incorrect namespace is left.
With that in mind i think `<?php ?>` is the safer route to go until the directive finds better support.

Reply:
No problem at all, let's stick with `<?php ?>`.